core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/TextReplace.java

```diff
@@ -3,29 +3,42 @@
 import io.metadew.iesi.script.execution.instruction.data.DataInstruction;
 
 import java.text.MessageFormat;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class TextReplace implements DataInstruction {
 
+    private final static String FIRST_OPERATOR = "text";
+    private final static String SECOND_OPERATOR = "start";
+    private final static String THIRD_OPERATOR = "end";
+    private final static Pattern THREE_ARGUMENTS_PATTERN = Pattern.compile("\"(?<" +FIRST_OPERATOR + ">.+)\"," + "\\s\"\\s*(?<" + SECOND_OPERATOR + ">.+)\"," +
+            "\\s\"\\s*(?<" + THIRD_OPERATOR + ">.+)\"");
+
+    private final static Pattern TWO_ARGUMENTS_PATTERN = Pattern.compile("\"(?<" +FIRST_OPERATOR + ">.+)\"," + "\\s\"\\s*(?<" +
+            SECOND_OPERATOR + ">.+)\"");
```

Comment:
pattern should be something like 
\\s*\"(?\<text>.+)\"\\s*,\\s*\"(?\<start>.+)\"\\s*

please retest with this format